src/main/java/com/selfxdsd/todocli/JsonTodo.java

```diff
@@ -15,8 +17,12 @@
     public JsonTodo(final Todo todo) {
         super(Json.createObjectBuilder()
                 .add("id", todo.getID())
-                .add("author", todo.getAuthor())
-                .add("timestamp", todo.getTimestamp())
+                .add("author", Optional.ofNullable(todo.getAuthor())
+                    .<JsonValue>map(Json::createValue)
+                    .orElse(JsonValue.NULL))
+                .add("timestamp", Optional.ofNullable(todo.getTimestamp())
+                    .<JsonValue>map(Json::createValue)
+                    .orElse(JsonValue.NULL))
```

Comment:
@amihaiemil For the time when is no git repository.

Reply:
@criske da, dar din cate am inteles din modificarile facute, parseru nu va gasi niciun todo daca folderul nu e un repo de git, sau? :D

Reply:
@amihaiemil daca nu e git, parserul va cauta direct in fisier si va exclude GIT_BLAME_PATTERN din regex. Acum cum e scris codul, parser-ul obtine fisierul prin git blame.

Reply:
@amihaiemil Codul asta l-am scris preventiv pentru atunci cand parser-ul va suporta atat folder cu git repo cat si fara. :D

Reply:
@criske 

> Acum cum e scris codul, parser-ul obtine fisierul prin git blame.

Exact, la asta ma refer. Daca folder-ul nu e un git repository, git blame nu va merge si nu va fi gasit niciun fisier, asa inteleg :D 

Dar e ok momentan, ca oricum nu folosim todo-finder-cli decat cu proiecte care sunt git. Voi deschide un ticket mai incolo.

Reply:
@amihaiemil vad ca a picat un test
Expected:
`"author":"Luka","timestamp":"2020-11-08 09:32:57 +0100"`
Actual
`"author":"amihaiemil","timestamp":"2020-11-09 19:26:02 +0200"`

Ciudat e ca la mine nu te vad pe tine in blame la todo start .

Reply:
@amihaiemil Ok am testat si pe linux cu `todo-finder-cli` la zi, si imi arata "Luka" in todo.json la fel ca si pe win.

Reply:
@criske si eu am buildutit local, si n-am avut probleme :D